ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ccm/cucushift-installer-rehearse-aws-ipi-ccm-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-ccm
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-ccm-provision
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-ccm-deprovision
+  documentation: |-
+    This is the workflow to trigger Prow's rehearsal test when submitting installer steps/chain/workflow
```

Comment:
we'd better add the supported versions in workflow documentation  as well.

Reply:
> we'd better add the supported versions in workflow documentation as well.

added, thanks